integration/spark2/src/main/scala/org/apache/carbondata/indexserver/DistributedPruneRDD.scala

```diff
@@ -49,7 +49,8 @@ private[indexserver] class DistributedPruneRDD(@transient private val ss: SparkS
     dataMapFormat: DistributableDataMapFormat)
   extends CarbonRDD[(String, ExtendedBlocklet)](ss, Nil) {
 
-  val executorsList: Set[String] = DistributionUtil.getNodeList(ss.sparkContext).toSet
+  @transient val executorsList: Map[String, Seq[String]] = DistributionUtil
```

Comment:
I don't think it is required to keep at class level, better move it to method level as you are using at one place.

Reply:
moved to internalGetPartitions